bats/tests/helpers/utils.bash

```diff
@@ -307,6 +307,32 @@ EOF
     fi
 }
 
+# create_file path/to/file <<< "contents"
+# Create a new file with the provided path; the contents of standard input will
+# be written to that file.  Analogous to `cat >$1`.  Will create any parent
+# directories.
+create_file() {
+    # On Windows, avoid creating files from within WSL; this leads to issues
```

Comment:
Should there be a `local file=$1`, as `$1` is referenced at least 4 times below.